sklearn/titanic/README.md

```diff
@@ -0,0 +1,138 @@
+## Introduction
+
+This sample code illustrates how to start a *Google Cloud Engine (GCE)* instance and train a model using scikit-learn on the instance. We will be using the [Titanic dataset](https://www.kaggle.com/c/titanic) from [Kaggle](https://www.kaggle.com/). The goal is to train a classifier to predict whether a given passenger on Titanic survived or not.
+
+## Prerequisites
+
+Before we start, we need to:
+
+* login to [Kaggle](https://www.kaggle.com) and download the Titanic dataset, mainly [train.csv](https://www.kaggle.com/c/titanic/download/train.csv).
+
+* have a [*GCP*](https://cloud.google.com/) account, and [download](https://cloud.google.com/sdk/), install, and [configure](https://cloud.google.com/sdk/gcloud/reference/config/) the *Google Cloud SDK* on our local computer.
+
+* [create a project](https://cloud.google.com/sdk/gcloud/reference/projects/create) on *GCP* and set the project, region, and zone properties as instructed [here](https://cloud.google.com/sdk/gcloud/reference/config/set).
+
+* set these environment variables on the local computer:
+	* $MY_INSTANCE: the name of the new instance (e.g. ```export $MY_INSTANCE=titanic_trainer```).
+	* $MY_BUCKET: the bucket name to be created on *Google Cloud Storage (GCS)* (e.g. ```export MY_BUCKET=titanic_bucket```).
+	* $MY_FOLDER: the folder name to be used in the bucket (e.g. ``` export MY_FOLDER=titanic_folder```).
+
+## Steps:
+We will create a [VM instance](https://cloud.google.com/compute/docs/instances/), configure it, and use it to train a model. While some of the steps can be done using the web portal [here](https://pantheon.corp.google.com/compute/instances), we will try to accomplish this using the *SDK* and mainly the [*gcloud*](https://cloud.google.com/sdk/gcloud/) command. Unless otherwise specified as a comment, all the commands are to be run locally.
+
+##### 1. Create a Google Compute Engine Instance
+First, we should create a *GCE* instance that we can use to train our model on. In this case, we can rely on many of the default arguments:
+
+```bash
+gcloud compute instances create $MY_INSTANCE
+```
+which creates an instance of type *n1-standard-1* (1 CPU, 3.75GB of RAM), with a *Debian* disk image and no scopes. This is fine as long as we need not the instance access *GCS*. Otherwise, we will have to provide the right scope:
```

Comment:
"This is fine as long as we need not the instance access *GCS*."

Not sure, what this line means.